### DISCUSSION TITLE
DataView not work 10.2.1

### DISCUSSION THREAD

**Discussion by mafin1799** [primefaces/community#986] [PrimeReact] [answered]
![image](https://github.com/primefaces/community/assets/90977148/d2684312-a90d-49ce-837c-3cfcf7b6d0dc)
dataview component is not working. errors in examples with dock

**Comment by mafin1799** [comment]
Event types are dead in the latest version

**Reply by melloware** [reply, marked as answer]
Works for me: https://stackblitz.com/edit/vitejs-vite-tfmmnp?file=src%2FApp.tsx
